src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -117,6 +122,26 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
             }
         }
 
+        public List<int> GetPortsToIgnoreFromAnnotations(V1ServiceList serviceNameSpace) {
+            var portsToIgnore = new List<int>();
+            List<V1Service> portsList = serviceNameSpace.Items.Where(item => item.Metadata?.Annotations?.ContainsKey(ServiceAnnotations) ?? false).ToList();
+            portsList.ForEach(port => {
+                if (port.Metadata?.Annotations?.TryGetValue(ServiceAnnotations, out string ports) ?? false) {
+                    if (ports.Length > 0) {
+                        try {
+                            // ports e.g. "460, 543"
+                            portsToIgnore = ports.Split(",").Select(Int32.Parse)?.ToList();

```

Comment:
I dont think this works if there is more than one service in the namespace. It will get overriden instead of append. Also I think we need to keep track of what ports for each service. for example if there are 3 services in the namespace and the config of service A says ignore port 80, serivce B says ignore port 300, service C says ignore port 3000. Then serivce A configures port 80 and 300, for service A we should only ignore port 80 not both.